spec/integrations/disk_spec.rb

```diff
@@ -0,0 +1,56 @@
+describe 'datadog::disk' do
+  expected_yaml = <<-EOF
+  init_config: {}
+
+  instances:
+    - use_mount: 'yes'
+      excluded_filesystems:
+        - tmpfs
+      excluded_disks:
+        - /dev/sda1
+        - /dev/sda2
+      excluded_disk_re: '/dev/sde.*'
+      tag_by_filesystem: 'no'
+      excluded_mountpoint_re: '/mnt/somebody-elses-problem.*'
+      all_partitions: 'no'
+  EOF
+
+  cached(:chef_run) do
+    ChefSpec::SoloRunner.new(step_into: ['datadog_monitor']) do |node|
+      node.automatic['languages'] = { 'python' => { 'version' => '2.7.2' } }
+      node.set['datadog'] = {
+        api_key: 'someapikey',
+        disk: {
+          instances: [
+            {
+              use_mount: 'yes',
+              excluded_filesystems: ['tmpfs'],
+              excluded_disks: ['/dev/sda1', '/dev/sda2'],
+              excluded_disk_re: '/dev/sde.*',
+              tag_by_filesystem: 'no',
+              excluded_mountpoint_re: '/mnt/somebody-elses-problem.*',
+              all_partitions: 'no'
```

Comment:
Ah, right. I think I had quoted those because the 'yes' and 'no' string attributes were getting converted to true/false booleans in the file and the spec was failing. Using true/false directly definitely removes confusion here.